pkgs/development/python-modules/tuya-device-sharing-sdk/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, paho-mqtt
+, cryptography
+}:
+let
+  pname = "tuya-device-sharing-sdk";
+  version = "0.2.0";
+in
+buildPythonPackage {
+  inherit pname version;
```

```suggestion
  inherit pname version;
  pyproject = true;
```

Enables the more modern PEP517 build process. Requires configuring

```nix
build-system = [
	setuptools
]
```

Please right before `dependencies`.